lib/browser/storage.js

```diff
@@ -22,17 +22,23 @@ try {
 
 export const canStoreURLs = hasStorage;
 
-export function setItem(key, value) {
-  if (!hasStorage) return;
-  return localStorage.setItem(key, value);
-}
+class LocalStorage {
+  setItem(key, value, cb) {
+    if (!hasStorage) return cb();
+    cb(null, localStorage.setItem(key, value));
+  }
 
-export function getItem(key) {
-  if (!hasStorage) return;
-  return localStorage.getItem(key);
+  getItem(key, cb) {
+    if (!hasStorage) return cb();
+    cb(null, localStorage.getItem(key));
+  }
+
+  removeItem(key, cb) {
+    if (!hasStorage) return cb();
+    cb(null, localStorage.removeItem(key));
+  }
 }
 
-export function removeItem(key) {
-  if (!hasStorage) return;
-  return localStorage.removeItem(key);
+export function getStorage() {
+  return new LocalStorage();
```

Comment:
Instead of checking for `!hasStorage` in every method of LocalStorage we can now check for it inside `getSource`, i.e.:

```
getStorage() {
  if(!hasStorage) return null;
  return new LocalStorage();
}
```

Reply:
Addressed in https://github.com/tus/tus-js-client/pull/152.